src/hapi/pipelines/utilities/operational_presence.py

```diff
@@ -0,0 +1,201 @@
+"""Functions specific to the operational presence theme."""
+import hxl
+
+from logging import getLogger
+from typing import Dict
+
+from sqlalchemy.orm import Session
+
+from hdx.scraper.base_scraper import BaseScraper
+from hapi.pipelines.database.db_operational_presence import DBOperationalPresence
+from hapi.pipelines.utilities.admins import (
+    Admins,
+    get_admin2_to_admin1_connector_code,
+)
+# from hapi.pipelines.utilities.metadata import Metadata
+from hapi.pipelines.utilities.org import Org
+from hapi.pipelines.utilities.orgtype import OrgType
+from hapi.pipelines.utilities.sector import Sector
+
+logger = getLogger(__name__)
+
+
+class OperationalPresence(BaseScraper):
+
+    def __init__(self, session: Session, datasetinfo: Dict):
+        super().__init__(
+            "operational_presence",
+            datasetinfo,
+            dict(),
+        )
+        self._session = session
+        self.data = []
+
+    def run(self):
+        reader = self.get_reader()
+        reader.read_hdx_metadata(self.datasetinfo)
+        data = hxl.data(self.datasetinfo["url"])
+        if "#org" not in data.tags or "#sector" not in data.tags or (
+                "#adm1" not in data.tags and "#adm2" not in data.tags):
```

Comment:
I tried to write it to be a bit more general.  Since the data can vary so much country to country, right now you don't need to pass in the exact tags for org, sector, or acronym, it just finds them on its own.  I could switch and make the configuration much more explicit - not sure which one is the right approach!  @mcarans could you weigh in?

Reply:
I changed the structure a bit to include the headers in the yaml so removed these!